src/Ormolu/Fixity.hs

```diff
@@ -160,6 +161,35 @@ packageToPopularity :: HashMap String Int
 HackageInfo packageToOps packageToPopularity =
   fromJust . A.decodeStrict $ hackageInfoFile
 
+-- | List of packages shipped with GHC, for which the download count from
+-- Hackage does not reflect their high popularity.
+-- See https://github.com/tweag/ormolu/pull/830#issuecomment-986609572.
+-- "base" is not is this list, because it is already whitelisted
+-- by buildFixityMap'.
+bootPackages :: [String]
```

Comment:
Can this be an internal detail of the extraction script?

Reply:
I don't think that it is a good idea to put the boot packages override in the extraction script:

- we moved the "base" whitelisting + colon definition from the extraction script to `Fixity.hs`, so to stay consistent, I think we should keep any other kind of override in `Fixity.hs`
- In the extraction script, we don't know the threshold that's going to be used to build the fixity map, and thus, we can't know how many "virtual" downloads we should add to the download count of boot packages so that they become the reference for the operators they define